### ISSUE TITLE
Additional icons/watermarks on photos in different views

### ISSUE THREAD

**Issue by jimmcslim** [photoprism/photoprism#314]
A few ideas for additional icons that could be associated with a photo, either as a watermark icon in the card & mosaic views (and which maybe only appear when hovering, as opposed to the stack icon which is always visible?):

* A mobile phone or camera icon, to indicate whether the photo was taken by camera or phone; suggesting the camera one is likely, maybe, to be higher quality. I don't know if there is an external database that can be used to lookup from EXIF manufacturer/model tags to identify the 'class' of device that it is - this might be something that the user maintains themselves beyond some easy guesses e.g. certain manufacturers only make cameras; Canon, Nikon, and others only make mobile phones; Nokia, Apple (ok, yeah there was the Apple QuickTake!). A few manufacturers, e.g. Samsung, Sony, have made both.
* A panorama icon, for photos that have an aspect ratio suggesting they are panoramic (defined as an aspect ratio above 16:9?). In the mosaic and list views it isn't obvious, in the card view you can see the dimensions but it might still be handy.
* I'm sure I'll come up with a few more!

**Comment by ravetam** [comment]
In Lightroom, you can easily see if a photo has a RAW image next to the JPG. Add a RAW icon/label?

**Comment by lastzero** [comment]
> In Lightroom, you can easily see if a photo has a RAW image next to the JPG. Add a RAW icon/label?

Possible. In Lightroom it makes more sense though as it comes with a raw converter and image editor. Seeing a RAW in PhotoPrism wouldn't give you any additional options or benefit, just the information.

**Comment by ravetam** [comment]
> 
> 
> > In Lightroom, you can easily see if a photo has a RAW image next to the JPG. Add a RAW icon/label?
> 
> Possible. In Lightroom it makes more sense though as it comes with a raw converter and image editor. Seeing a RAW in PhotoPrism wouldn't give you any additional options or benefit, just the information.

Well, its a nice to have for me to save my storage space. I took a lot of photos with my camera. I don't usually delete JPEG from my vacation photo unless it is out of focus, but it the photo is average, I will delete the RAW file. Having indexed all my photos from the past into Photoprism, its nice being able to indicate which photo has a RAW and which one's not. At the moment, I have to go to the "Files" tab to find that out.

**Comment by lastzero** [comment]
A camera icon is now shown on RAW photos!

Since the list of icons and file types is open, I consider this issue an epic and not something that can be done at once. We can leave it here as inspiration and reminder.

**Comment by lastzero** [comment]
Build status: https://travis-ci.org/github/photoprism/photoprism/builds/689610465

**Comment by ravetam** [comment]
Just got the new docker image and after re-indexed my photos, I can now see the camera (RAW) icon on photos that has both RAW and JPG images. Looks great!

**Comment by lastzero** [comment]
Material Design sadly doesn't come with a real RAW logo like e.g. Canon uses it: https://material.io/resources/icons/?style=baseline

Glad the "camera" is OK for the moment :)

**Comment by ravetam** [comment]
Looked at material icons and seems like they also have panorama icon. For multiple icons, they should stack.

Some sample code for panorama. Aspect ratio above 2 or 3 should work.
```
<v-btn v-else-if="photo.FileAspectRatio >= 3" :ripple="false"
      icon flat large absolute class="p-photo-merged opacity-75"
      @click.stop.prevent="openPhoto(index, true)" title="Panorama">
       <v-icon color="white" class="action-burst">panorama_horizontal</v-icon>
</v-btn>
```

Example with multiple icons. Doesn't work well though on white background
![Sample](https://user-images.githubusercontent.com/3193896/82638471-895f1400-9c39-11ea-9a9f-1d2853ed6d7d.jpg)

**Comment by lastzero** [comment]
@ravetam Yes, bright backgrounds are not perfect yet. Glad most photos are not white.

Note that FileAspectRatio refers to the primary file in this case, all viewable files are listed in photo.Files (like additional JPEGs or Videos). So it's theoretically possible that there is a square and panorama JPEG version of the same RAW file.

**Comment by 0xmex** [comment]
I also second this.  An option to add a watermark for downloaded photos as a shared folder option would be amazing to prevent usage on a set of imaged meant as selection pool for the client.
